pyomo/contrib/mindtpy/MindtPy.py

```diff
@@ -300,6 +305,12 @@ def solve(self, model, **kwds):
         if config.max_slack == 0.0:
             config.add_slack = False
 
+        if config.strategy == "GOA":
+            config.add_integer_cuts = True
+            config.add_slack = True
+            config.use_mcpp = True
+            config.integer_to_binary = True
```

Comment:
This is only since we have not implemented the Integer cuts for integer variables, right? The more I think about it, the more I would encourage to change the name of `integer_cuts` to `nogood_cuts`